src/components/PageContentMain/PageContentMain.js

```diff
@@ -9,9 +9,10 @@ export default function PageContentMain({
   heading,
 }) {
   const classNames = composeClassNames(
-    className || styles.PageContentMain,
-    styles[`PageContentMain--${variant}`] || styles[`PageContentMain--default`]
+    styles.PageContentMain,
+    className,
+    styles[`PageContentMain__${variant}`] || styles.PageContentMain__default
   );
 
-  return <article className={classNames}>{children}</article>;
+  return <main className={classNames}>{children}</main>;
```

Comment:
Ah, OK. You did the main here. Will this work if we have full width components in the main?

Reply:
No, I have to make an adjustment for that.